--- a/core/rend/gles/gles.cpp	2019-06-27 12:54:23.000000000 +0200
+++ b/core/rend/gles/gles.cpp	2019-07-09 21:12:06.287108385 +0200
@@ -372,7 +372,8 @@
    	shader->trilinear = trilinear;
    	CompilePipelineShader(shader);
    }
-
+ 	glcache.UseProgram(shader->program); //FIX RPI4
+        ShaderUniforms.Set(shader);
    return shader;
 }
 
@@ -575,7 +576,7 @@
 		s->fog_clamp_max = -1;
 	}
 
-   ShaderUniforms.Set(s);
+   //ShaderUniforms.Set(s);
 
 	return glIsProgram(s->program)==GL_TRUE;
 }

